src/Explore.Cli/Program.cs

```diff
@@ -348,10 +366,36 @@ internal static async Task ExportSpaces(string exploreCookie, bool? verboseOutpu
                 ExploreSpaces = spacesToExport
             };
 
+            // set the file name if provided
+            var fileName = string.IsNullOrEmpty(exportFileName) ? "ExploreSpaces.json" : $"{exportFileName}.json";
+
+            // set the export path if provided
+            string filePath;
+            if (!string.IsNullOrEmpty(exportPath))
+            {
+                // check if the exportPath is an absolute path
+                if (!Path.IsPathRooted(exportPath))
+                {
+                    // if not, make it relative to the current directory
+                    exportPath = Path.Combine(Environment.CurrentDirectory, exportPath);
+                }
+
+                // combine the path and filename
+                filePath = Path.Combine(exportPath, fileName);
+
+                if (!Directory.Exists(exportPath))

```

Comment:
We should also check/handle permissions issues where we may not be able to create the directory or write the file to the specified directory. See the importSpaces function for similar logic